package/libyui-ncurses-pkg.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Mon Jun 25 09:58:08 CEST 2018 - aschnell@suse.com
+
+- use long long instead of int for free disk space with MiB base
```

Comment:
Please include the bug reference.
The comment describes the implementation but does not tell the user the effect or the reason for the change. How about "Fixed an erroneous warning about not enough space on exabyte-sized disks" ?